task/mgmt-gateway/src/update/sp.rs

```diff
@@ -54,7 +78,8 @@ impl SpUpdate {
                 ResponseError::OtherComponentUpdateInProgress(component)
             })?;
 
-        // TODO handle auxflash updates
+        // Can we handle an auxflash update?
+        #[cfg(not(feature = "auxflash"))]
         if update.aux_flash_size > 0 {
```

Comment:
IMHO, this would be more clear as
```rust
if update.aux_flash_size > 0 && !cfg!(feature = "auxflash") {
```
or
```rust
if update.aux_flash_size > 0 && cfg!(not(feature = "auxflash")) {
```